tools/distpartitioning/dist_lookup.py

```diff
@@ -296,6 +296,9 @@ def get_shuffle_nids(self, global_nids, my_global_nids, my_shuffle_global_nids):
 
         # Send the shuffle-global-nids to their respective ranks.
         mapped_global_nids = alltoallv_cpu(self.rank, self.world_size, shuffle_nids_list)
+        for idx in range(len(mapped_global_nids)):
```

Comment:
why not use `for idx in mapped_global_nids:` ?

Reply:
If we use `for item in mapped_global_nids:`, then we can check if item is None or not. But this will not solve our purpose. Here, we are removing `None` in the mapped_global_nids (which is a list of tensors and some can be None as well). 

we can also rewrite the code as follows: 
for idx, item in enumerate(mapped_global_nids):
  if item == None: 
     mapped_global_nids[idx] = torch.empty((0,), dtype=torch.int64)
mapped_global_nids = torch.cat(mapped_global_nids). 

Either way I am fine with the code. 
Please let me know in case you think that this for-loop needs to change. I will change it accordingly.